src/middleware/express/make-http-request.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HttpRequest } from '../../http-request';
+// Types-only import.
+import * as express from 'express';
+
+export function makeHttpRequestData(req: express.Request, res: express.Response, latencyMilliseconds: number): HttpRequest {
```

Comment:
<3 this is much more convenient than expecting users to build this logic in their apps